### PR TITLE
api gardening

### PR DESCRIPTION
**Pull request by marxian**

### CODE REVIEW

**Comment by marxian** [comment]
#### 📝Changed routes:


| [**/api**](https://gbptm-next-lwdtgqqc2-neontribe.vercel.app/api) | [**/**](https://gbptm-next-lwdtgqqc2-neontribe.vercel.app/) |
|:-:|:-:|
|<a href="https://gbptm-next-lwdtgqqc2-neontribe.vercel.app/api">Screenshot of /api</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/5VTBH1IlsLsWQZAu1L9RHN0yKupktyMpk-1W4KTFdFnGlNnmS3HOvjzLCZUfs0HyHy3D55PJiOakJBYPRgvJcw.png">(view full size)</a> |<a href="https://gbptm-next-lwdtgqqc2-neontribe.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/J1mywu8lGaaQtSoPQdMzoHM5oHfrX66JmTYppDf_diy2L5ExtEzoDiiCRQZehF3kNrmLJ-Pcn-PA0S1dHiXbyA.png">(view full size)</a> |







Commit 9c6d6720b4aaed6c30eb1dc3077d9f86387bd406 (https://gbptm-next-lwdtgqqc2-neontribe.vercel.app).

**Comment by ob6160** [comment]
nice one, looks like it loads toilets now :) this one good to go in?

File: package.json [hunk 1]
```diff
@@ -51,7 +50,7 @@
     "@material-ui/icons": "4.11.2",
     "@rehooks/component-size": "1.0.3",
     "@styled-system/prop-types": "5.1.5",
-    "apollo-server": "2.24.1",
+    "apollo-server-micro": "2.25.2",
```
Comment (ob6160) [inline package.json:53]: please excuse my ignorance - just wondering what `apollo-server-micro` doing that `apollo-server` wasn't able to do?

File: src/pages/index.tsx [hunk 2]
```diff
@@ -116,7 +116,6 @@ const HomePage = ({ initialPosition, ...props }) => {
   );
 
   const isLooPage = router.pathname === '/loos/[id]';
-    console.log("AA", isLooPage, router.pathname)
```
Comment (ob6160) [inline src/pages/index.tsx:119]: thanks missed this

**Review by ob6160 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (marxian) [reply package.json:53]: Vercel/Next.js uses micro to front the api functions, so it is the preferred integration.

File: src/pages/index.tsx (hunk 2 above)
Comment (marxian) [reply src/pages/index.tsx:119]: Natch

File: package.json (hunk 1 above)
Comment (ob6160) [reply package.json:53]: ah right right sounds good

File: src/pages/index.tsx (hunk 2 above)
Comment (ob6160) [reply src/pages/index.tsx:119]: 🙏

**Pull request merged**